create_crafting_db.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import json
+import sqlite3
+from pathlib import Path
+
+
+def build_database(json_path: Path, db_path: Path) -> None:
+    data = json.loads(json_path.read_text())
+    conn = sqlite3.connect(db_path)
+    cur = conn.cursor()
+
+    cur.executescript(
+        """
+        PRAGMA journal_mode = WAL;
+        PRAGMA synchronous = NORMAL;
+
+        CREATE TABLE IF NOT EXISTS items (
+            id INTEGER PRIMARY KEY,
```

Comment:
I'm not experienced in SQL, but as I understand `INTEGER` means Int32. It doesn't fit, items IDs are expected to be in range of UInt64 (`BIGINT UNSIGNED`), by using Int32 you will most likely get overflow on some cargo items.